packages/zundo/README.md

```diff
@@ -221,25 +218,6 @@ const useStoreA = create<StoreState>(
 );
 ```
 
-#### **Cool-off period**
-
-`handleSet?: (handleSet: StoreApi<TState>['setState']) => StoreApi<TState>['setState']`
-
-Sometimes multiple state changes might happen in a short amount of time and you only want to store one change in history. To do so, we can utilize the `handleSet` callback to set a timeout to prevent new changes from being stored in history. This can be used with something like `lodash.throttle` or `debounce`. This a way to provide middleware to the temporal store's setter function.
```

Comment:
I need to add this back to the config, but under the `wrap temporal store` options.